README.md

```diff
@@ -53,12 +53,12 @@ See our [Code of Conduct](CODE_OF_CONDUCT.md) and our [Contributor Guide](CONTRI
+      <td align="center" valign="top" width="14.28%"><a href="https://evelinag.com/"><img src="https://avatars.githubusercontent.com/u/5541162?v=4?s=100" width="100px;" alt="Evelina Gabasova"/><br /><sub><b>Evelina Gabasova</b></sub></a><br /><a href="#content-evelinag" title="Content">🖋</a></td>
       <td align="center" valign="top" width="14.28%"><a href="https://github.com/fedenanni"><img src="https://avatars.githubusercontent.com/u/8415204?v=4?s=100" width="100px;" alt="Federico Nanni"/><br /><sub><b>Federico Nanni</b></sub></a><br /><a href="https://github.com/alan-turing-institute/data-safe-haven/commits?author=fedenanni" title="Code">💻</a> <a href="https://github.com/alan-turing-institute/data-safe-haven/issues?q=author%3Afedenanni" title="Bug reports">🐛</a> <a href="https://github.com/alan-turing-institute/data-safe-haven/commits?author=fedenanni" title="Documentation">📖</a> <a href="#ideas-fedenanni" title="Ideas, Planning, & Feedback">🤔</a></td>
       <td align="center" valign="top" width="14.28%"><a href="https://github.com/fkiraly"><img src="https://avatars.githubusercontent.com/u/7985502?v=4?s=100" width="100px;" alt="Franz Király"/><br /><sub><b>Franz Király</b></sub></a><br /><a href="#content-fkiraly" title="Content">🖋</a></td>
       <td align="center" valign="top" width="14.28%"><a href="https://github.com/gn5"><img src="https://avatars.githubusercontent.com/u/50482094?v=4?s=100" width="100px;" alt="Guillaume Noell"/><br /><sub><b>Guillaume Noell</b></sub></a><br /><a href="https://github.com/alan-turing-institute/data-safe-haven/commits?author=gn5" title="Documentation">📖</a> <a href="https://github.com/alan-turing-institute/data-safe-haven/issues?q=author%3Agn5" title="Bug reports">🐛</a> <a href="#ideas-gn5" title="Ideas, Planning, & Feedback">🤔</a></td>
-      <td align="center" valign="top" width="14.28%"><a href="http://helen.st/cv"><img src="https://avatars.githubusercontent.com/u/217966?v=4?s=100" width="100px;" alt="Helen Sherwood-Taylor"/><br /><sub><b>Helen Sherwood-Taylor</b></sub></a><br /><a href="#ideas-helenst" title="Ideas, Planning, & Feedback">🤔</a> <a href="#content-helenst" title="Content">🖋</a></td>
+      <td align="center" valign="top" width="14.28%"><a href="https://helen.st/cv"><img src="https://avatars.githubusercontent.com/u/217966?v=4?s=100" width="100px;" alt="Helen Sherwood-Taylor"/><br /><sub><b>Helen Sherwood-Taylor</b></sub></a><br /><a href="#ideas-helenst" title="Ideas, Planning, & Feedback">🤔</a> <a href="#content-helenst" title="Content">🖋</a></td>
```

Comment:
This probably won't work (if there is no valid SSL cert).

Manual changes like this will always be overwritten when you run `all-contributors` too.
It would be better to exclude these urls from the check.